spec/unit/cookbook/metadata_spec.rb

```diff
@@ -684,9 +788,14 @@
       metadata.attribute "bizspark/has_login",
         :display_name => "You have nothing"
       metadata.version "1.2.3"
+      metadata.chef_version ">= 11.14.2", "< 11.18.10"
+      metadata.chef_version ">= 12.2.1", "< 12.5.1"
+      metadata.ohai_version ">= 7.1.0", "< 7.5.0"
+      metadata.ohai_version ">= 8.0.1", "< 8.6.0"
     end
 
     it "should produce the same output from to_json and Chef::JSONCompat" do
+      # XXX: fairly certain this is testing ruby method dispatch
```

Comment:
FWIW, the test comes from https://github.com/chef/chef/commit/1c7b4629240c49e02c482da6de1a6a2cc574304e